bsip-0043.md

```diff
@@ -61,4 +61,6 @@ The new modification - market fee sharing - will allow to bring in new clients f
 This document is placed in the public domain.
 
 # See Also
-https://github.com/bitshares/bitshares-core/issues/1268
\ No newline at end of file
+https://github.com/bitshares/bitshares-core/issues/1268
+
+https://github.com/bitshares/bitshares-core/pull/1419
```

Comment:
Please make a separate pull request for this change

Reply:
done